src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -0,0 +1,275 @@
+      ::gr::Tags::SqrtDetSpatialMetric<DataVector>,
+      ::gr::Tags::SpacetimeMetric<SpatialDim, Frame, DataVector>, ::Tags::Time,
+      Tags::GaugeHRollOnStartTime, Tags::GaugeHRollOnTimeWindow,
+      ::Tags::Coordinates<SpatialDim, Frame>,
+      Tags::GaugeHSpatialWeightDecayWidth<Frame>>;
+
+  static typename db::item_type<Tags::GaugeH<SpatialDim, Frame>> function(
+      const typename db::item_type<Tags::InitialGaugeH<SpatialDim, Frame>>&
+          gauge_h_init,
+      const Scalar<DataVector>& lapse,
+      const tnsr::I<DataVector, SpatialDim, Frame>& shift,
+      const Scalar<DataVector>& sqrt_det_spatial_metric,
+      const tnsr::aa<DataVector, SpatialDim, Frame>& spacetime_metric,
+      double time, double t_start, double sigma_t,
+      const tnsr::I<DataVector, SpatialDim, Frame>& coords,
+      double sigma_r) noexcept;
+};
+
+template <size_t SpatialDim, typename Frame>
+void compute_damped_harmonic_h(
+    gsl::not_null<typename db::item_type<Tags::GaugeH<SpatialDim, Frame>>*>
+        gauge_h,
+    const typename db::item_type<Tags::InitialGaugeH<SpatialDim, Frame>>&
+        gauge_h_init,
+    const Scalar<DataVector>& lapse,
+    const tnsr::I<DataVector, SpatialDim, Frame>& shift,
+    const Scalar<DataVector>& sqrt_det_spatial_metric,
+    const tnsr::aa<DataVector, SpatialDim, Frame>& spacetime_metric,
+    double time, const tnsr::I<DataVector, SpatialDim, Frame>& coords,
+    // Scaling coeffs in front of each term
+    double amp_coef_L1, double amp_coef_L2, double amp_coef_S,
+    // exponents
+    int exp_L1, int exp_L2, int exp_S,
+    // roll on function parameters for lapse / shift terms
+    double t_start_h_init, double sigma_t_h_init, double t_start_L1,
+    double sigma_t_L1, double t_start_L2, double sigma_t_L2, double t_start_S,
+    double sigma_t_S,
+    // weight function
+    double sigma_r) noexcept;
+// @}
+
+// @{
+/*!
+ * \ingroup GeneralizedHarmonic
+ * \brief Spacetime derivatives of the damped harmonic gauge source function.
+ *
+ * \details Compute spacetime derivatives, i.e. \f$\partial_a H_b\f$, of the
+ * damped hamornic source function H. From above, we rewrite it as:
+ * \f{align*}
+ * \partial_a H_b =& \partial_a T_1 + \partial_a T_2 + \partial_a T_3 \\
+ * H_a =& T_1 + T_2 + T_3,
+ * \f}
+ *
+ * where:
+ *
+ * \f{align*}
+ * T_1 =& [1 - R_{H_\mathrm{init}}(t)] H_a^\mathrm{init}, \\
+ * T_2 =& [\mu_{L1} \mathrm{log}(\sqrt{g}/N) + \mu_{L2} \mathrm{log}(1/N)] t_a,
+ * \\ T_3 =& - \mu_S g_{ai} N^i / N. \f}
+ *
+ * Derivation:-
+ * (**) For \f$ T_1 \f$, the derivs are:
+ * \f{align*}
+ * \partial_a T_1 = (1 - R_{H_\mathrm{init}}(t)) *
+ * \partial_a H_b^\mathrm{init}
+ *                - H_b^\mathrm{init} \partial_a R_{H_\mathrm{init}}.
+ * \f}
+ *
+ * (**) Write \f$ T_2 = (\mu_1 + \mu_2) t_b \f$. Then:
+ * \f{align*}
+ * \partial_a T_2 =& (\partial_a \mu_1 + \partial_a \mu_2) t_b \\
+ *               +& (\mu_1 + \mu_2) \partial_a t_b
+ * \f},
```

Comment:
this comma is rendered before the where rather than after the equation, should be inside the equation environment.